.github/workflows/scala3-compile.yml

```diff
@@ -16,17 +16,17 @@ jobs:
     strategy:
       matrix:
         command:
-          - akka-testkit/Test/compile akka-actor-tests/Test/compile
-          - akka-actor-testkit-typed/Test/compile akka-actor-typed-tests/Test/compile
-          - akka-bench-jmh/Test/compile
-          - akka-cluster/Test/compile akka-cluster-tools/Test/compile akka-cluster-typed/Test/compile akka-distributed-data/Test/compile akka-cluster-metrics/Test/compile akka-cluster-sharding/Test/compile akka-cluster-sharding-typed/Test/compile
-          - akka-discovery/Test/compile akka-coordination/Test/compile
-          - akka-persistence/Test/compile akka-persistence-shared/Test/compile akka-persistence-query/Test/compile akka-persistence-typed/Test/compile akka-persistence-testkit/Test/compile akka-persistence-tck/Test/compile akka-persistence-typed-tests/Test/compile
-          - akka-pki/Test/compile akka-slf4j/Test/compile
-          - akka-serialization-jackson/Test/compile
-          - akka-stream/Test/compile akka-stream-testkit/Test/compile akka-stream-tests/Test/compile akka-stream-typed/Test/compile
-          - akka-stream-tests-tck/Test/compile
-          - akka-remote/Test/compile akka-remote-tests/Test/compile akka-protobuf/Test/compile akka-protobuf-v3/Test/compile
+          - pekko-testkit/Test/compile akka-actor-tests/Test/compile
+          - pekko-actor-testkit-typed/Test/compile akka-actor-typed-tests/Test/compile
+          - pekko-bench-jmh/Test/compile
+          - pekko-cluster/Test/compile akka-cluster-tools/Test/compile akka-cluster-typed/Test/compile akka-distributed-data/Test/compile akka-cluster-metrics/Test/compile akka-cluster-sharding/Test/compile akka-cluster-sharding-typed/Test/compile
+          - pekko-discovery/Test/compile akka-coordination/Test/compile
+          - pekko-persistence/Test/compile akka-persistence-shared/Test/compile akka-persistence-query/Test/compile akka-persistence-typed/Test/compile akka-persistence-testkit/Test/compile akka-persistence-tck/Test/compile akka-persistence-typed-tests/Test/compile
+          - pekko-pki/Test/compile akka-slf4j/Test/compile
+          - pekko-serialization-jackson/Test/compile
+          - pekko-stream/Test/compile akka-stream-testkit/Test/compile akka-stream-tests/Test/compile akka-stream-typed/Test/compile
+          - pekko-stream-tests-tck/Test/compile
```

Comment:
I have these changes in a PR that is in progress - but only renames the modules that the PR is renaming